### PR TITLE
Use same warning icons that jenkins.io is using

### PR DESCRIPTION
**Pull request by halkeye**
* Update makelayout to wrap script tags in helmet (so they work in dev mode)
* Replace custom warning icon with <ion-icon class="report" name="warning" />
* make `yarn dev` work toplevel cause i'm lazy

### CODE REVIEW

**Comment by halkeye** [comment]
I'm very very very excited for this, this means my idea of having a web component for header and footer is totally doable

File: plugins/plugin-site/src/components/PluginActiveWarnings.jsx [hunk 1]
```diff
@@ -13,7 +12,7 @@ function PluginActiveWarnings({securityWarnings}) {
     }
     return (
         <div className="alert alert-danger alert-with-icon">
-            <WarningsIcon />
+            <ion-icon class="report" name="warning" />
```
Comment (zbynek) [inline plugins/plugin-site/src/components/PluginActiveWarnings.jsx]:
```suggestion
            <ion-icon class="alert-icon" name="warning" />
```

File: plugins/plugin-site/src/components/PluginGovernanceStatus.jsx [hunk 2]
```diff
@@ -10,7 +9,7 @@ function PluginGovernanceStatus({plugin}) {
         if (id === 'adopt-this-plugin') {
             return (
                 <div className="alert alert-warning alert-with-icon" key={id}>
-                    <WarningsIcon />
+                    <ion-icon class="report" name="warning" />
```
Comment (zbynek) [inline plugins/plugin-site/src/components/PluginGovernanceStatus.jsx]:
```suggestion
                    <ion-icon class="alert-icon" name="warning" />
```

File: plugins/plugin-site/src/components/PluginGovernanceStatus.jsx [hunk 3]
```diff
@@ -20,7 +19,7 @@ function PluginGovernanceStatus({plugin}) {
         } else if (id === 'deprecated') {
             return (
                 <div className="alert alert-warning alert-with-icon" key={id}>
-                    <WarningsIcon />
+                    <ion-icon class="report" name="warning" />
```
Comment (zbynek) [inline plugins/plugin-site/src/components/PluginGovernanceStatus.jsx]:
```suggestion
                    <ion-icon class="alert-icon" name="warning" />
```

**Review by zbynek — CHANGES_REQUESTED** [review]
Thanks, looks good, just two styling issues.
 * On jenkins.io the icon in footer is before the link and is separated from the text by a space, would be nice to do the same here
 * Within the alert boxes the icon should have `alert-icon` class instead of `report` to have the correct color and style (see suggestions)

File: plugins/plugin-site/src/templates/tombstone.jsx [hunk 4]
```diff
@@ -22,7 +21,7 @@ function TombstonePage({data: {suspendedPlugin: plugin}}) {
                     </div>
                 </div>
                 <div className="alert alert-warning alert-with-icon">
-                    <WarningsIcon/>
+                    <ion-icon class="report" name="warning" />
```
Comment (zbynek) [inline plugins/plugin-site/src/templates/tombstone.jsx]:
```suggestion
                    <ion-icon class="alert-icon" name="warning" />
```

**Comment by halkeye** [comment]
Oh good call, i sorta just did a search and replace.

File: plugins/plugin-site/src/components/ReportAProblem.jsx [hunk 5]
```diff
@@ -8,6 +8,7 @@ function ReportAProblem({reportProblemTitle, reportProblemUrl, reportProblemRela
         <p className="box">
             <a href={reportProblemUrl.includes('://') ? reportProblemUrl : pluginSiteReportUrl} title={title}>
                 Report a problem
```
Comment (halkeye) [inline plugins/plugin-site/src/components/ReportAProblem.jsx]: I'll try to flip this when I'm at a computer next

**Pull request merged**